src/model/tableNode.ts

```diff
@@ -9,7 +9,8 @@ import { INode } from "./INode";
 
 export class TableNode implements INode {
     constructor(private readonly host: string, private readonly user: string, private readonly password: string,

```

Comment:
Also update mysql.createConnection in this file

Reply:
Not sure what you mean here, there is no mysql.createConnection call in this file?

Reply:
It's here: https://github.com/formulahendry/vscode-mysql/blob/5d28e182fb9be7bb0c5fd49e1b96a172760e0c98/src/model/tableNode.ts#L27

Reply:
Oh well, that was changed from another pull request just recently.. ill make the changes